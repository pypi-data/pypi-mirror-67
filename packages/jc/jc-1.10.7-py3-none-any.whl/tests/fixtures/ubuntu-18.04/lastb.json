[{"user": "UNKNOWN", "tty": "ttyS0", "hostname": null, "login": "Fri Feb 14 02:06", "logout": "02:06", "duration": "00:00"}]
