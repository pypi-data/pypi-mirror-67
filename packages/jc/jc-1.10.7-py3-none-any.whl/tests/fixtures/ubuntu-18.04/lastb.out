UNKNOWN  ttyS0                         Fri Feb 14 02:06 - 02:06  (00:00)

btmp begins Fri Feb 14 02:06:41 2020
